app/views/layouts/_sub_menu.html.erb

```diff
@@ -12,7 +12,8 @@
         controller.controller_name == 'annotation_categories' ||
         controller.controller_name == 'grade_entry_forms' ||
         controller.controller_name == 'automated_tests' ||
-        controller.controller_name == 'tags' %>
+        controller.controller_name == 'tags' ||
```

Comment:
Please revert the changes in this file, and the index file you created below. (Again, this is later.)